v2/scripts/generate_docs.sh

```diff
@@ -11,3 +11,7 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
 else
     sed -i 's|contracts/|protocol/contracts/|g' docs/src/index.md
 fi
+
+find docs/src/ -type f -name "*.md" -exec sed -i.bak -E 's|(https://github.com/zeta-chain/protocol-contracts/blob/)[^/]+/|\1main/v2/|g' {} +
```

Comment:
If I recall correctly you may need to add `''` so this works on linux, but I could be wrong

```suggestion
find docs/src/ -type f -name "*.md" -exec sed -i.bak '' -E 's|(https://github.com/zeta-chain/protocol-contracts/blob/)[^/]+/|\1main/v2/|g' {} +

```

Reply:
Good catch! Once this is merged, I'll add docs generation check as part of CI, and if it fails, I'll introduce this fix to make it work both on macOS and Linux.